assets/src/graph/cdGlobalServices.graphql

```diff
@@ -63,9 +113,14 @@ query GlobalServices($first: Int, $after: String, $projectId: ID) {
   }
 }
 
-query GetServiceData($serviceId: ID!, $first: Int, $after: String) {
+query GetGlobalService($serviceId: ID!) {
   globalService(id: $serviceId) {
     ...GlobalService
+  }
+}
+
+query GetGlobalServiceServices($serviceId: ID!, $first: Int, $after: String) {
+  globalService(id: $serviceId) {
     services(first: $first, after: $after) {
```

Comment:
still want to include the top-level service id here, apollo will have issues caching it otherwise

see https://github.com/pluralsh/console/pull/1103/